src/GitExtensions.PluginTemplate/Plugin.cs

```diff
@@ -43,14 +43,21 @@ public override void Unregister(IGitUICommands gitUiCommands)
             
         }
 
+        ///// <summary>
+        ///// See: https://github.com/gitextensions/gitextensions.plugintemplate/wiki/GitPluginBase#public-virtual-ienumerableisetting-getsettings
+        ///// </summary>
+        //public override IEnumerable<ISetting> GetSettings()
+        //{
+        //    // Uncomment and fill if your plugin have settings that the user could configure
+        //}
 
         /// <summary>
         /// See: https://github.com/gitextensions/gitextensions.plugintemplate/wiki/GitPluginBase#public-abstract-bool-executegituieventargs-args
         /// </summary>
         public override bool Execute(GitUIEventArgs gitUIEventArgs)
         {
             MessageBox.Show(gitUIEventArgs.OwnerForm, "Hello from the Plugin Template.", "Git Extensions");
-            return true;
+            return false; //Must return true if the revision grid should be refreshed after the execution of the plugin. false, otherwise.

```

Comment:
This comment belongs to the xml-doc

Reply:
I know, I wrote the xml doc just before 😉

That join your previous comment on the lack of doc here and using just the wiki links.

I put the comment here because that's the typical information that no one will look in the doc, so I think there is more chance that the developper will see it.

Perhaps add it in the summary?

Reply:
Return information belongs to `return` section, e.g.
```
<summary>
 ....
</summary>
  Returns <see langword="true"/> if the revision grid should be refreshed after the execution of the plugin;
  otherwise <see langword="false"/>.
<returns>
</returns>
```